src/components/m-form-group/_preview.html

```diff
@@ -34,6 +34,11 @@
                   info="Caramels liquorice ice cream gingerbread gummi bears. Lemon drops cookie wafer cake. Sweet roll sesame snaps chocolate bar bear claw lemon drops sugar plum cupcake candy canes icing.">
     <axa-input value="Pre-filled content" name="test-name" error="Some error!"></axa-input>
   </axa-form-group>
+  <axa-form-group label="Test Label"
```

Comment:
we could discuss to rename `label` to `legend` for consistency, though this would be breaking change 🤔